yaml_test.go

```diff
@@ -271,6 +271,10 @@ func TestJSONToYAML(t *testing.T) {
 			`{"t":null}`,
 			"t: null\n",
 			nil,
+		},{
```

Comment:
I think this line break lint tests

Reply:
@akram can you format the files in a follow up PR?

Reply:
sure @liggitt

Reply:
here it is https://github.com/kubernetes-sigs/yaml/pull/63